### ISSUE TITLE
android delete command fails

### ISSUE THREAD

**Issue by taukakao** [Vanilla-OS/vanilla-system-operator#143]
When you try, it just gives you: Error Code 127 and exits without doing anything

**Comment by TiramisuOS** [comment]
Yes, I am also experiencing the same thing and I have tried many times but still failed, Hopefully there is a solution

**Comment by taukakao** [comment]
@Supergamerx3000 The apps are in ~/.local/share/applications
